pkg/transformer/kubernetes/k8sutils.go

```diff
@@ -282,6 +282,27 @@ func (k *Kubernetes) CreateService(name string, service kobject.ServiceConfig, o
 	return svc
 }
 
+// CreateHeadlessService creates a k8s headless service
+func (k *Kubernetes) CreateHeadlessService(name string, service kobject.ServiceConfig, objects []runtime.Object) *api.Service {
```

Comment:
What is a 'Headless' service? *I* know what it means, but perhaps adding a comment to the kubernetes documentation may help.